tests/functional/sync_reset_password.js

```diff
@@ -160,8 +158,7 @@ define([
         .then(testElementExists('#fxa-complete-reset-password-header'))
         .then(fillOutCompleteResetPassword(PASSWORD, PASSWORD))
 
-        .then(testElementExists('#fxa-reset-password-complete-header'))
-        .then(testElementTextInclude('.account-ready-service', 'Firefox Sync'));
```

Comment:
Even though the service is not dynamically injected, could you do a check for `testElementExists('.account-ready-service')`?